neuralmonkey/decoding_function.py

```diff
@@ -1,59 +1,93 @@
 import tensorflow as tf
```

Comment:
CTC je tohle: http://www.cs.toronto.edu/~graves/icml_2006.pdf
Používalo se na to speech recognition a handwriting recognition.

Bude hnusnej, ale spraví se to pak, že se zase půlka vycestuje do toho bezattentionovýho dekodéru.